experimental/LieAlgebras/src/WeylGroup.jl

```diff
@@ -257,3 +257,97 @@ function isomorphism(::Type{PermGroup}, W::WeylGroup; set_properties::Bool=true)
 
   return MapFromFunc(W, G, iso, isoinv)
 end
+
+@doc raw"""
+    map_word(w::WeylGroupElem, genimgs::Vector; init = nothing)
+
+If `init` is `nothing` and `word(w) = [UInt(`$i_1$`), ..., UInt(`$i_n$`)]`, then return the product $R_1 R_2 \cdots R_n$
```

Comment:
I would omit the `UInt` for clarity
```suggestion
If `init` is `nothing` and `word(w) = [`$i_1$`, ..., `$i_n$`]`, then return the product $R_1 R_2 \cdots R_n$
```

Reply:
I thought about that as well, but the trade-off between clarity and correctness wasn't evident for me. Anyway, I'll remove it.